ci3.9.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - xarray=2022.10.0
   - xarray-beam=0.3.1
   - ecmwf-api-client=1.6.3
-  - flake8=5.0.4
+  - black=22.12.0
```

Comment:
In addition to adding the dependencies here, let's also add them to the `setup.py` file at the root of the project in the `test` section (see the list `all_test_requirements`).

Reply:
Sure in `setup.py` file under ,
```
test_requirements = [
    "pytype==2021.11.29",
```
```diff
-  "flake8",
+  "pyink",
```
```
    "pytest",
    "pytest-subtests",
.
.
.
```
I can change it I think.